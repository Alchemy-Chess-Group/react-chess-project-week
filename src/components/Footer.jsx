export default function Footer() {
  return <div>alchemy code lab</div>;
}
