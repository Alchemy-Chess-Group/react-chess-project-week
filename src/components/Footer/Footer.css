footer {
  background-color: #418dbd;
  color: rgb(235, 222, 203);
  padding: auto;
  position: absolute;
  bottom: 0;
  width: 100%;
  height: 50px;
  display: flex;
  justify-content: flex-end;
}
