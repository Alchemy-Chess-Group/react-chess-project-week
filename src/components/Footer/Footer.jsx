import style from './Footer.css';

export default function Footer() {
  return (
    <footer className={style.footer}>
      <h5>alchemy code lab</h5>
    </footer>
  );
}
