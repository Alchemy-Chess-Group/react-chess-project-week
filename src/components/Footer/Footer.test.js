import { render } from '@testing-library/react';
import Footer from './Footer';

test('Test the footer renders', () => {
  const { container } = render(<Footer />);

  expect(container).toMatchSnapshot();
});
