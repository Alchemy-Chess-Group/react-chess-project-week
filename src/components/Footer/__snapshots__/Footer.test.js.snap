// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the footer renders 1`] = `
<div>
  <footer
    class="footer"
  >
    <h5>
      alchemy code lab
    </h5>
  </footer>
</div>
`;
